channels-rke2.yaml

```diff
@@ -504,3 +552,27 @@ releases:
       rke2-coredns:
         repo: rancher-rke2-charts
         version: 1.17.000
+  - version: v1.23.6-rc2+rke2r1
+    minChannelServerVersion: v2.6.4-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs: &serverArgs-v1-23-6-rc2-rke2r1
+      <<: *serverArgs-v1-23-4-rke2r2
+    agentArgs: &agentArgs-v1-23-6-rc2-rke2r1
+      <<: *agentArgs-v1-23-4-rke2r2
+    charts: &charts-v1-23-6-rc2-rke2r1
+      <<: *charts-v1-23-5-rke2r1
```

Comment:
The server and agent args should also pull from v1-23-5-rke2r1 otherwise they will miss some of the changes that were present from those.